Makefile

```diff
@@ -13,6 +13,8 @@ APALACHE_PROTO := apalache/shai/src/main/protobuf
 # See this issue details: https://github.com/protocolbuffers/protobuf/issues/1491
 CHAI_PROTO := proto/chai
 
+.PHONY: apalache
```

Comment:
Shall we also include this previously defined target?

```suggestion
.PHONY: apalache update-grpc
```

Reply:
Oh good catch, thanks!